Lesson1/src/com/task4/Solution2.java

```diff
@@ -0,0 +1,40 @@
+package com.task4;
+
+import java.util.Scanner;
+
+/**
+ * Определить число, полученное выписыванием в обратном порядке цифр
+ * любого 4-х значного натурального числа n.
+ * Решение второе.
+ */
+public class Solution2 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Введите четыре цифры вашего числа: ");
```

Comment:
Not clear that I should you space between numbers)
if you just enter 4567 - application looks freezed